src/style.css

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use any rule inside `*` selector except for `border-box`.
Reset margin and padding directly where you need.